sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -914,14 +898,16 @@ def process_bundle(self,
                      expected_outputs,  # type: DataOutput
                      fired_timers,  # type: Mapping[Tuple[str, str], PartitionableBuffer]
                      expected_output_timers  # type: Dict[str, Dict[str, str]]
+                     dry_run=False
```

Comment:
This should be the default, we shouldn't have to pass it.

Reply:
Done